pkg/ddevapp/drupal.go

```diff
@@ -369,7 +369,6 @@ func drupalConfigOverrideAction(app *DdevApp) error {
 		app.Database = DatabaseDesc{Type: nodeps.MariaDB, Version: nodeps.MariaDB104}
 	case "9":
 		app.PHPVersion = nodeps.PHP81
-		app.Database = DatabaseDesc{Type: nodeps.MariaDB, Version: nodeps.MariaDB104}
```

Comment:
good idea removing that one. with the current default 10.4 i ran into the following error testing with @penyaskitos artifact: 

```
$> ddev config --update
You are reconfiguring the project at /Users/rkoller/Sites/d9.
The existing configuration will be updated and replaced.
Configuring a 'drupal' project with docroot 'web' at /Users/rkoller/Sites/d9/web
unable to configure project d9 with database type mariadb:10.11 because that database type does not match the current actual database. Please change your database type back to mariadb:10.11 and start again, export, delete, and then change configuration and start. To get back to existing type use 'ddev config --database=mariadb:10.11', and you can try a migration with 'ddev debug migrate-database mariadb:10.4' see docs at https://ddev.readthedocs.io/en/stable/users/extend/database-types/```

Reply:
@penyaskito is going to make it not override existing database config for this exact reason.

Reply:
yep. so removing 10.4 for d9 is a good call. just wanted to echo that.

Reply:
Is this because 10.4 is the global default?
If not, I'll keep this and just avoid overriding the DB in the other PR. But for new D9 sites might make sense.

Reply:
No, mariadb:10.11 is the new default.  But D9 doesn't really care what version of mariadb it has.